yarn.lock

```diff
@@ -1384,6 +1384,16 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
+"@fluentui/theme@^1.5.1":
```

Comment:
We need to ensure we're specifying dependencies in a way that doesn't accidentally pull in extra versions of packages that exist in master... May need to mess with `resolutions` in `package.json`? Or ideally figure out how to fix it "properly."

Reply:
Same issue for styling and utilities. I think we should try to find a "proper" fix for this if possible since it's an issue our consumers will hit too. (Or at least figure out some guidance which we use internally and share out.)